posthog/hogql/database/schema/web_analytics_s3.py

```diff
@@ -0,0 +1,100 @@
+from posthog.hogql.database.models import (
+    DatabaseField,
+    IntegerDatabaseField,
+    DateTimeDatabaseField,
+    FieldOrTable,
+)
+from posthog.hogql.database.s3_table import S3Table
+from django.conf import settings
+from posthog.settings.object_storage import (
+    OBJECT_STORAGE_BUCKET,
+    OBJECT_STORAGE_ENDPOINT,
+    OBJECT_STORAGE_ACCESS_KEY_ID,
+    OBJECT_STORAGE_SECRET_ACCESS_KEY,
+    OBJECT_STORAGE_PREAGGREGATED_WEB_ANALYTICS_FOLDER,
+)
+
+
+def get_s3_url(team_id: int, table_name: str) -> str:
+    if settings.DEBUG:
+        s3_endpoint = "http://objectstorage:19000"
+        bucket = "posthog"
+        key = f"{OBJECT_STORAGE_PREAGGREGATED_WEB_ANALYTICS_FOLDER}/{table_name}"
+        return f"{s3_endpoint}/{bucket}/{key}"
+    else:
+        if OBJECT_STORAGE_ENDPOINT:
+            endpoint = OBJECT_STORAGE_ENDPOINT.replace("localhost:19000", "objectstorage:19000")
+            base_url = f"{endpoint}/{OBJECT_STORAGE_BUCKET}"
+        else:
+            base_url = f"https://{OBJECT_STORAGE_BUCKET}.s3.amazonaws.com"
+
+        return f"{base_url}/{OBJECT_STORAGE_PREAGGREGATED_WEB_ANALYTICS_FOLDER}/{table_name}"
+
+
+def get_s3_web_stats_structure() -> str:
+    return """
+        period_bucket DateTime,
+        team_id UInt64,
+        persons_uniq_state AggregateFunction(uniq, UUID),
+        sessions_uniq_state AggregateFunction(uniq, String),
+        pageviews_count_state AggregateFunction(sum, UInt64)
+    """
+
+
+def get_s3_web_bounces_structure() -> str:
+    return """
+        period_bucket DateTime,
+        team_id UInt64,
+        persons_uniq_state AggregateFunction(uniq, UUID),
+        sessions_uniq_state AggregateFunction(uniq, String),
+        pageviews_count_state AggregateFunction(sum, UInt64),
+        bounces_count_state AggregateFunction(sum, UInt64),
+        total_session_duration_state AggregateFunction(sum, Int64)
+    """
+
+
+WEB_STATS_S3_FIELDS: dict[str, FieldOrTable] = {
```

Comment:
nit: we could grab those from the other schema and build the specific one here: https://github.com/PostHog/posthog/blob/6c3c4d35145e5ccc8c76f352549516f0c0dea73e/posthog/hogql/database/schema/web_analytics_preaggregated.py#L10-L73